Added IANA considerations and compile-time recommendation wording. (#6)
tlswg/draft-ietf-tls-ech-keylogfile: draft-ietf-tls-ech-keylogfile.md

- Access to the ECH_SECRET record in the SSLKEYLOGFILE allows the attacker to decrypt the ECH extension and thereby reveal the content of the ClientHello message, including the payload of the Server Name Indication (SNI) extension.

- Access to the HPKE-established shared secret introduces a potential attack surface against the HPKE library since access to this keying material is normally not available otherwise.

Implementers MUST take measures to prevent unauthorized access to the SSLKEYLOGFILE text file.

> As per the SSLKEYLOGFILE specification {{!I-D.ietf-tls-keylogfile}}, this extension is intended for use in environments where TLS protects only test data. While the access it provides to TLS connections can be valuable for debugging during development, this mechanism MUST NOT be used in production environments. To minimize the risk of accidental activation in production, implementers SHOULD incorporate appropriate compile-time controls.

Reviewer: ```suggestion
As per the SSLKEYLOGFILE specification {{!I-D.ietf-tls-keylogfile}}, this extension is intended for use in environments where TLS protects only test data. While the access it provides to TLS connections can be valuable for debugging during development, this mechanism MUST NOT be used in production environments. To minimize the risk of accidental activation in production, implementers SHOULD incorporate appropriate compile-time controls.
```

> According to SSLKEYLOGFILE specification {{!I-D.ietf-tls-keylogfile}}, this extension is intended for use in systems where TLS only protects test data. While the access this information provides to TLS connections can be useful for diagnosing problems during development, this mechanism MUST NOT be used in a production environment. To minimize risks of accidentally enabling this mechanism in production environments implementers SHOULD provide relevant compile-time controls.

Reviewer: ```suggestion
As per the SSLKEYLOGFILE specification {{!I-D.ietf-tls-keylogfile}}, this extension is intended for use in environments where TLS protects only test data. While the access it provides to TLS connections can be valuable for debugging during development, this mechanism MUST NOT be used in production environments. To minimize the risk of accidental activation in production, implementers SHOULD incorporate appropriate compile-time controls.
```

# IANA Considerations

> IANA is requested to create a new registry "SSLKEYLOGFILE labels", within the existing "Transport Layer Security (TLS) Parameters" registry page.

Reviewer: ```suggestion
IANA is requested to create a new registry "SSLKEYLOGFILE Labels", within the existing "Transport Layer Security (TLS) Parameters" registry page.
```
